test/modern_bpf/test_suites/syscall_exit_suite/pipe_x.cpp

```diff
@@ -0,0 +1,107 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_pipe
+
+#if defined(__NR_fstat) && defined(__NR_close)
+
+TEST(SyscallExit, pipeX_success)
+{
+	auto evt_test = new event_test(__NR_pipe, EXIT_EVENT);
+
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL ===========================*/
+
+	int32_t pipefd[2];
+	assert_syscall_state(SYSCALL_SUCCESS, "pipe", syscall(__NR_pipe, pipefd), NOT_EQUAL, -1);
+
+	struct stat file_stat;
+	assert_syscall_state(SYSCALL_SUCCESS, "fstat", syscall(__NR_fstat, pipefd[0], &file_stat), NOT_EQUAL, -1);
```

Comment:
syscall assertion likely depends on discussion [here](https://github.com/falcosecurity/libs/pull/514#discussion_r934427907)

Reply:
I do not have a strong opinion here... so it is fine to do right here or post-pone and do a focused review at the end of the porting phase.